tests/cases/fourslash/completionsClassMemberImportTypeNodeParameter2.ts

```diff
@@ -0,0 +1,31 @@
+/// <reference path="fourslash.ts" />
+
+// @module: nodenext
+
+// @FileName: /index.d.ts
+//// export declare class Cls {
+////   method(
+////     param: import("./doesntexist.js").Foo,
+////   ): import("./doesntexist.js").Foo;
+//// }
+////
+//// export declare class Derived extends Cls {
+////   /*1*/
+//// }
+
+verify.completions({
+  marker: "1",
+  includes: [
+    {
+      name: "method",
+      insertText: `method(param: import("./doesntexist.js").Foo);`,
```

Comment:
This is weird but not anything new. Return types nodes with errors are omitted based on `NodeBuilderFlags.SuppressAnyReturnType`. No mechanism like this for parameters is used.